mainwindow.py

```diff
@@ -405,91 +409,6 @@ def __init__(self, parent, kicad_provider=KicadProvider()):
         rotation.SetSortable(True)
         side.SetSortable(True)
 
```

Comment:
ahh k, could always rebase and fixup the previous change with this one if that makes sense with your git-fu

Reply:
No idea what you mean by that 😅